python-avd/pyavd/_eos_cli_config_gen/j2templates/documentation/arp.j2

```diff
@@ -20,18 +20,12 @@
 Global ARP timeout: {{ arp.aging.timeout_default }}
 {%     endif %}
 {%     if arp.static_entries is arista.avd.defined %}
-{# TODO - remove when dropping support for ansible 2.12.6 and its jinja2 hack #}
-{%         for entry in arp.static_entries %}
-{%             if entry.vrf is not arista.avd.defined %}
-{%                 do entry.update({"vrf": "default"}) %}
-{%             endif %}
-{%         endfor %}
 
 #### ARP Static Entries
 
 | VRF | IPv4 address | MAC address |
 | --- | ------------ | ----------- |
-{%         for vrf, entries in arp.static_entries | groupby("vrf") | arista.avd.natural_sort %}
+{%         for vrf, entries in arp.static_entries | arista.avd.default([]) | groupby("vrf", default="default") | arista.avd.natural_sort %}
```

Comment:
No need for this change. We already check for static_entries in line 22

Reply:
notice the `default` keyword in groupby which is what was missing from the hacky patch in ansible.
Though I don't need the default indeed will fix this